cmd/bisync/cmd.go

```diff
@@ -0,0 +1,224 @@
+// Package bisync implements bisync
+// Copyright (c) 2017-2020 Chris Nelson
+package bisync
+
+import (
+	"context"
+	"crypto/md5"
+	"encoding/hex"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/rclone/rclone/cmd"
+	"github.com/rclone/rclone/cmd/bisync/bilib"
+	"github.com/rclone/rclone/fs"
+	"github.com/rclone/rclone/fs/config"
+	"github.com/rclone/rclone/fs/config/flags"
+	"github.com/rclone/rclone/fs/filter"
+	"github.com/rclone/rclone/fs/hash"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+// Options keep bisync options
+type Options struct {
+	Resync          bool
+	CheckAccess     bool
+	CheckFilename   string
+	CheckSync       CheckSyncMode
+	RemoveEmptyDirs bool
+	MaxDelete       int // percentage from 0 to 100
+	Force           bool
+	FiltersFile     string
+	Workdir         string
+	DryRun          bool
+	NoCleanup       bool
+	SaveQueues      bool // save extra debugging files (test only flag)
+}
+
+// Default values
+const (
+	DefaultMaxDelete     int    = 50
+	DefaultCheckFilename string = "RCLONE_TEST"
```

Comment:
@ncw 

Probably `.rclonetest` would be a better name to be in line with `.rclonelink` (https://rclone.org/flags)